docs/content/architecture/pgadmin4.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "pgAdmin 4"
+date:
+draft: false
+weight: 900
+---
+
+![pgAdmin 4 Query](/images/pgadmin4-query.png)
+
+[pgAdmin 4](https://www.pgadmin.org/) is a popular graphical user interface that
+makes it easy to work with PostgreSQL databases from both a desktop or web-based
+client. With its ability to manage and orchestrate changes for PostgreSQL users,
+the PostgreSQL Operator is a natural partner to keep a pgAdmin 4 environment
+synchronized with a PostgreSQL environment.
+
+The PostgreSQL Operator lets you deploy a pgAdmin 4 environment alongside a
+PostgreSQL cluster and keeps users' database credentials synchronized. You can
+simply log into pgAdmin 4 with your PostgreSQL username and password and
+immediately have access to your databases.
+
+## Deploying pgAdmin 4
+
+Let's use the PostgreSQL cluster we created in the [quickstart]({{< relref "quickstart/_index.md" >}})
+as an example. If you don't have the `hippo` example cluster running, follow those steps.
+
+After the PostgreSQL cluster becomes ready, you can create a pgAdmin 4 deployment. To add the
+deployment to your cluster, you must add the following fields to your spec. 
+
+```
+  userInterface:
+    pgAdmin:
+      image: registry.developers.crunchydata.com/crunchydata/crunchy-pgadmin4:centos8-13.5-4.5.5
+      dataVolumeClaimSpec:
+        accessModes:
+        - "ReadWriteOnce"
+        resources:
+          requests:
+            storage: 1Gi
+```
+
+This creates a pgAdmin 4 deployment unique to this PostgreSQL cluster and synchronizes the
+PostgreSQL user information. To access pgAdmin 4, you can set up a port-forward to the Service,
```

Comment:
⛏️ Why "Service" rather than "service" -- is that a pattern we have?

Reply:
I would assume because it is a Kubernetes object, but again I copied from the old docs.

Reply:
You know, I just double-checked, and it looks like we do have that pattern of capitalizing Service.

Reply:
Except in docs/content/tutorial/connect-cluster.md line 52, where it appears lowercased.

Is that worth changing?